docs/content/en/docs/implementing/slo/_index.md

```diff
@@ -0,0 +1,109 @@
+title: Analysis
+description: Understand Analyses in Keptn and how to use them
+weight: 150
+cascade:
+  type: docs
+  currentversion: main
+---
+
+The Keptn Metrics Operator implements an SLO/SLI feature set inspired by Keptn v1 under the name of Analysis.
+With an Analysis Definition you can specify multiple Service Level Objectives that will be evaluated in your Analysis.
+At the end of the Analysis the status returns whether your objective failed or passed.
+
+The Analysis result is exposed as an OpenTelemetry metric and can be displayed on dashboard tools, such as Grafana.
+
+Keptn v1 users may use [SLO](https://github.com/keptn/lifecycle-toolkit/blob/main/metrics-operator/converter/slo_converter.md#slo-converter) and [SLI converters](https://github.com/keptn/lifecycle-toolkit/blob/main/metrics-operator/converter/sli_converter.md#sli-converter) to migrate towards Keptn Analysis. 
+
+## Keptn Analysis basics
+
+A Keptn Analysis is implemented with three resources:
+
+* [Analysis](https://lifecycle.keptn.sh/docs/crd-ref/metrics/v1alpha3/#analysis) --
+  define the specific configurations and the analysis to report
+* [AnalysisDefinition](https://lifecycle.keptn.sh/docs/crd-ref/metrics/v1alpha3/#analysisdefinition) --
+  define the list of SLOs for an analysis
+* [AnalysisValueTemplate](https://lifecycle.keptn.sh/docs/crd-ref/metrics/v1alpha3/#analysisvaluetemplate) --
+  define the SLI: the KeptnMetricsProvider and the Query to perform for each SLO
+
+### Define Analysis, Analysis Definition and AnalysisValueTemplate
+
+An Analysis customizes the templates defined inside an AnalysisDefinition by adding configurations such as:
+* a Timeframe that specifies the range for the corresponding query in the AnalysisValueTemplate
+* a map of key/value pairs that can be used to substitute placeholders in the AnalysisValueTemplate
+
+An AnalysisDefinition contains a list of objectives to satisfy.
+Each of these objectives:
+* specifies failure or warning target criteria,
+* specifies whether the objective is a key objective (its failure would fail the analysis)
+* indicates the weight of the objective on the overall analysis
+* refers to an AnalysisValueTemplate that contains the SLIs, defining the data provider from which to gather the data and how to compute the analysis
+
+In each AnalysisValueTemplate we store the query for the analysis of the SLI. You must define a
+[KeptnMetricsProvider](../../yaml-crd-ref/metricsprovider.md) resource
+for each instance of each data provider you are using.
+The template refers to that provider and queries it.
+
+Let's consider the following Analysis:
+
+{{< embed path="/metrics-operator/config/samples/metrics_v1alpha3_analysis.yaml" >}}
+
+This CR sets up the timeframe we are interested in
+as between 5am and 10am of the 5th of May 2023 ,
+and adds a few specific key-value pair that will be substituted in the query
+for instance the query could contain a {{.nodename }} and this value will be substituted by test
```

Comment:
```suggestion
For instance, the query could contain a "{{.nodename }}" and this value will be substituted by test
```
or
```suggestion
For instance, the query could contain a `{{.nodename }}` and this value will be substituted by test
```

based on which render better 🤔

Reply:
since in the example later, the query contains the {{.nodename}}, I'd remove the "could"